backend/tests/unit/services/test_document_tombstone_filter.py

```
@@ -0,0 +1,100 @@
+    await db.flush()
+    ws = Workspace(name="WS", description="", owner_id=user.id)
+    db.add(ws)
+    await db.flush()
+    return ws
+
+
+async def _make_doc(db: AsyncSession, workspace_id, *, deleted: bool = False) -> Document:
+    doc = Document(
+        workspace_id=workspace_id,
+        filename="f.pdf",
+        file_path="/tmp/f.pdf",
```

Ruff (0.15.14)

[error] Probable insecure usage of temporary file or directory: "/tmp/f.pdf"

(S108)